pkg/model/resource/resource.go

```diff
@@ -78,10 +137,94 @@ func (r Resource) Replacer() *strings.Replacer {
 	var replacements []string
 
 	replacements = append(replacements, wrapKey("group"), r.Group)
-	replacements = append(replacements, wrapKey("group-package-name"), r.GroupPackageName)
 	replacements = append(replacements, wrapKey("version"), r.Version)
 	replacements = append(replacements, wrapKey("kind"), strings.ToLower(r.Kind))
 	replacements = append(replacements, wrapKey("plural"), strings.ToLower(r.Plural))
+	replacements = append(replacements, wrapKey("package-name"), r.PackageName())
 
 	return strings.NewReplacer(replacements...)
 }
+
+// API holds the information related to the golang type definition and the CRD.
+type API struct {
+	// Version holds the CustomResourceDefinition API version used for the resource.
+	Version string `json:"crdVersion,omitempty"`
```

Comment:
```suggestion
	CrdVersion string `json:"crdVersion,omitempty"`
```

Reply:
Same as in https://github.com/kubernetes-sigs/kubebuilder/pull/1869#discussion_r540080054